### PR TITLE
Get auto join rooms

### PR DESCRIPTION
**Pull request by aliahmed922**
New Api for user to get joined rooms

### CODE REVIEW

File: lib/hipchat/api_version.rb [hunk 1]
```diff
@@ -302,6 +302,15 @@ def view_config
         }[version]
       end
 
+      def user_room_config
+        {
+          'v2' => {
```
Comment (rberrelleza) [inline lib/hipchat/api_version.rb]: Can you raise an InvalidApiVersion exception when the api is set to v1 (see history_config for an example)

File: lib/hipchat/api_version.rb [hunk 2]
```diff
@@ -302,6 +302,15 @@ def view_config
         }[version]
       end
 
+      def user_room_config
```
Comment (rberrelleza) [inline lib/hipchat/api_version.rb]: I think the name is misleading. How about 'user_joined_rooms' or something similar?

Comment (aliahmed922) [inline lib/hipchat/api_version.rb]: Sure sir, Definitely your suggestion.

File: lib/hipchat/api_version.rb (hunk 1 above)
Comment (aliahmed922) [inline lib/hipchat/api_version.rb]: Ahh, I missed that I was in a hurry, since I gotta use this part. Let me do this.

**Comment by aliahmed922** [comment]
@rberrelleza Is there any thing left to change for this PR to get merged?

**Comment by rberrelleza** [comment]
Thanks for your contribution @aliahmed922 !

**Comment by aliahmed922** [comment]
@rberrelleza Mention Not

**Pull request merged**